cmd/config.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/odo/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+// configurationCmd represents the app command
+var configurationCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Modifies configuration settings",
+	Long:  `Modifies Odo specific configuration settings within the config file.`,
+	Example: fmt.Sprintf("%s\n%s\n",
+		configurationGetCmd.Example,
+		configurationSetCmd.Example),
+	Aliases: []string{"configuration"},
+	// 'odo config' is the same as 'odo config get'
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 1 && args[0] != "get" && args[0] != "set" {
+			return fmt.Errorf("Unknown command, use set or get")
+		}
+		return nil
+	}, Run: func(cmd *cobra.Command, args []string) {
+		if len(args) > 0 && args[0] == "set" {
+			configurationSetCmd.Run(cmd, args)
+		} else if len(args) > 0 && args[0] == "get" {
+			configurationGetCmd.Run(cmd, args)
+		} else {
+			configurationGetCmd.Run(cmd, args)
+		}
+	},
+}
+
+var configurationSetCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set a value in odo config file",
+	Long:  "Set an individual value in the Odo configuration file",
+	Example: `  # Set a configuration value
+   odo config set UpdateNotification false
+
+   Available Parameters: UpdateNotification
+ 	`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 2 {
+			return fmt.Errorf("Please provide a parameter name and value")
+		} else if len(args) > 2 {
+			return fmt.Errorf("Only one value per parameter is allowed")
+		} else {
+			return nil
+		}
+	}, Run: func(cmd *cobra.Command, args []string) {
+		cfg, err := config.New()
+		if err != nil {
+			errors.Wrapf(err, ": unable to set configuration")
+		}
+		value, err := strconv.ParseBool(args[1])
+		if err != nil {
+			errors.Wrapf(err, ": unable to set configuration")
+		}
+		cfg.SetConfiguration(strings.ToLower(args[0]), value)
+	},
+}
+
+var configurationGetCmd = &cobra.Command{
+	Use:   "get",
+	Short: "View current value",
+	Long:  "View current configuration",
+	Example: `  # Get the current configuration
+   odo config get
+ 	`,
+	Args: cobra.ExactArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+		cfg, err := config.New()
+		if err != nil {
+			fmt.Println(err, ": unable to get configuration")
+		}
+		w := tabwriter.NewWriter(os.Stdout, 5, 2, 2, ' ', tabwriter.TabIndent)
+		fmt.Fprintln(w, "Parameter", "\t", "CurrentValue")
```

Comment:
Let's make this all CAPS. To highlight that it is table header and to match the style that we are using in other lists.